core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionEvaluator.scala

```diff
@@ -78,26 +77,7 @@ class ExpressionEvaluator(slickEvaluator: SlickExpressionEvaluator, val rootEnti
     )
     */
   def evalFinalAttribute(workspaceContext: Workspace, expression: String)(implicit executionContext: ExecutionContext): ReadWriteAction[Map[String, Try[Iterable[AttributeValue]]]] = {
-    import slickEvaluator.parser.driver.api._
-
-    /*
-      Evaluate each attribute reference expression using the Slick Evaluator. It returns sequence of tuples of
-      attribute reference expressions and their slick evaluated value wrapped in a single DBIO action
-      For our example:
-        input: Set(this.bam, this.index)
-        output: ReadWriteAction(Seq(
-          ("this.bam", Map("101" -> Try(Seq(AttributeString("gs://abc"))), "102" -> Try(Seq(AttributeString("gs://def"))))),
-          ("this.index", Map("101" -> Try(Seq(AttributeNumber(123))), "102" -> Try(Seq(AttributeNumber(456)))))
-        ))
-     */
-    def evaluateAttributeReferences(attrRefs: Set[LookupExpression]): ReadWriteAction[Seq[(LookupExpression, Map[EntityName, Try[Iterable[AttributeValue]]])]] = {
-      val evaluatedLookupSeq: Seq[ReadWriteAction[(LookupExpression, Map[EntityName, Try[Iterable[AttributeValue]]])]] = attrRefs.toSeq.map { attrRef =>
-        slickEvaluator.evalFinalAttribute(workspaceContext, attrRef).map((attrRef, _))
-      }
-
-      // sequence Set[ReadWriteAction[T]] into single ReadWriteAction[Seq[T]]
-      DBIO.sequence(evaluatedLookupSeq)
-    }
+    import slickEvaluator.dataAccess.driver.api._
```

Comment:
This is one of the things we'll separate out later, right?  So I'm assuming we'll end up perhaps with a base or trait `ExpressionEvaluator` and then `LocalExpressionEvaluator` and `DataRepoExpressionEvaluator`?